packages/sandbox/package.json

```diff
@@ -1,16 +1,27 @@
 {
   "name": "@typescript/sandbox",
   "version": "0.1.0",
-  "main": "dist/index.js",
-  "private": true,
   "license": "MIT",
-  "types": "../../typescriptlang-org/static/js/sandbox/index.d.ts",
+  "author": "TypeScript team",
+  "homepage": "https://github.com/microsoft/TypeScript-Website",
+  "repository": {
+    "url": "https://github.com/microsoft/TypeScript-Website.git",
+    "directory": "packages/sandbox",
+    "type": "git"
+  },
+  "bugs": {
+    "url": "https://github.com/microsoft/TypeScript-Website/issues"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/sandbox.esm.js",
```

Comment:
Did you know that `module` is something bundlers made up out of thin air? 🙃 It’s ok to keep it, but we should use the new standard `exports` too:

```
"exports": {
  ".": {
    "types": "./dist/index.d.ts",
    "import": "./dist/sandbox.esm.js",
    "default": "./dist/index.js"
  }
}
```

This is assuming that the `index.d.ts` file works equally well for the ESM and CJS file, which is probably true if there’s no default export. If there are appreciable differences, there should be a separate `sandbox.esm.d.ts`. (By the way, why not just `sandbox.mjs` or `index.mjs` for that?)

Reply:
This is likely to keep it consistent with the rest of the packages in this repo, it's safe to say that tsdx has not kept up with the times: https://github.com/jaredpalmer/tsdx/releases

Reply:
I have to admit, my experience publishing packages to NPM is fairly limited! I didn't know about that key so good to know. Left the file extensions to keep things consistent per @orta's comment.